devito/core/gpu.py

```diff
@@ -7,7 +7,7 @@
 from devito.passes.equations import collect_derivatives, buffering
 from devito.passes.clusters import (Blocking, Lift, Streaming, Tasker, cire, cse,
                                     eliminate_arrays, extract_increments, factorize,
-                                    fuse, optimize_pows)
+                                    fuse, optimize_pows, Skewing)
```

Comment:
nitpicking: Skewing before Streaming (we do our best to keep alphabetical ordering)

Reply:
ok